src/views/partials/chat.hbs

```diff
@@ -0,0 +1,34 @@
+<div class="partial__chat__container">
```

Comment:
Instead of "partial__chat__container", just use "messages" (single name for the block)